### ISSUE TITLE
Output not minimized when using query parameter in filename

### ISSUE THREAD

**Issue by jazdw** [webpack/webpack#6716]
**Do you want to request a *feature* or report a *bug*?**
Bug

**What is the current behavior?**
If you use a query parameter in the file name as shown below the Javascript files will not be minimized by the UglifyJsPlugin plugin.

```
output: {
        filename: '[name].js?v=[chunkhash]',
}
```

This is due to the default test regex for the UglifyJsPlugin matching to the end of the line (`/\.js$/`). You can fix it by adding the following configuration.
```
    optimization: {
        minimizer: [
            new UglifyJsPlugin({
                test: /\.js($|\?)/
            })
        ]
    },
```

**If the current behavior is a bug, please provide the steps to reproduce.**
Use this config
```
output: {
        filename: '[name].js?v=[chunkhash]',
}
```

**What is the expected behavior?**
It should minimize the files.

**If this is a feature request, what is motivation or use case for changing the behavior?**

**Please mention other relevant information such as the browser version, Node.js version, webpack version, and Operating System.**
Webpack 4.1.1

**Comment by michael-ciniawsky** [comment]
Fixed by [#251](https://github.com/webpack-contrib/uglifyjs-webpack-plugin/pull/251) and released in [`v1.2.3`](https://github.com/webpack-contrib/uglifyjs-webpack-plugin/releases/tag/v1.2.3) 🎉

**Comment by jazdw** [comment]
Thanks @michael-ciniawsky sorry for the dupe. I made a comment on the pull request.
